requirements.txt

```diff
@@ -7,3 +7,4 @@ jinja2
 PyYAML
 cryptography
 packaging
+resolvelib >= 0.5.3  # New dependency resolver that pip>=20.3 uses
```

Comment:
@s-hertel @jborean93 https://github.com/sarugaku/resolvelib/issues/69 seems to indicate that 0.6 will have breaking changes so we probably need to cap the version too...